src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -37,6 +39,7 @@ export class ResultLayout extends Component {
    * The possible valid and supported layout.
    */
   public static validLayouts: ValidLayout[] = ['list', 'card', 'table'];
+
   /**
    * The current active layout.
```

Comment:
Since the multiple layout features aren't actually publicly available yet, I'm wondering whether we un-expose this from the public doc.

It seems to me like the getCurrentLayout() and changeLayout() public methods allow for safe manipulation of this attribute. What do you think?